iothub/device/samples/how to guides/DeviceReconnectionSample/DeviceReconnectionSample.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
+    <RootNamespace>Microsoft.Azure.Devices.Client.Samples</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.9.1" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.41.2" />
+
+    <!-- Working around a bug between DotNetty and .NET 5.0+ by specifying the exact version of DotNetty.Common that has the fix -->
+    <PackageReference Include="DotNetty.Common" Version="0.7.4" />
+  </ItemGroup>
+	
+  <ItemGroup Condition="'$(Configuration)' == 'Debug'">
```

Comment:
We can't have both a NuGet and a project reference (line 12) at the same time. Both need the same logic (configuration equals Debug or Release).